services/user.py

```diff
@@ -0,0 +1,51 @@
+from django.db import models
+from django.core.exceptions import ObjectDoesNotExist
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+User_model = get_user_model()
+
+
+def create_user(username: str,
+                password: str,
+                email: models.EmailField = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+
+    User_model.objects.create_user(username=username,
+                                   email=(email or ""),
+                                   first_name=(first_name or ""),
+                                   last_name=(last_name or ""),
+                                   password=password)
+
+
+def get_user(user_id: int) -> User | None:
+    try:
+        man = User_model.objects.get(id=user_id)
+    except ObjectDoesNotExist:
+        return None
```

Comment:
No need to explicitly return None here - let the exception happen and it may be try-excepted in place where this service is used